docs/docs/python.md

```diff
@@ -357,10 +351,10 @@ section.
 ### Client/Server Replicated Mode
 
 Using Tornado and
-[`PerspectiveTornadoHandler`](python.md#perspectivetornadohandler),
-as well as `Perspective`'s JavaScript library, we can set up "distributed"
-Perspective instances that allows multiple browser `perspective-viewer` clients
-to read from a common `perspective-python` server, as in the
+[`PerspectiveTornadoHandler`](python.md#perspectivetornadohandler), as well as
+`Perspective`'s JavaScript library, we can set up "distributed" Perspective
+instances that allows multiple browser `perspective-viewer` clients to read from
+a common `perspective-python` server, as in the
```

Comment:
This got picked up after auto formatting the markdown, we need to either auto format more regularly or otherwise enforce prettier on the markdown files to avoid these problems